verilog/tools/ls/verilog-language-server_test.cc

```diff
@@ -33,12 +36,27 @@ namespace {
 
 using namespace nlohmann;
```

Comment:
Don't use the generic namespace import, be specific in what you use (`using nlohmann::json`)